functions/src/index.ts

```diff
@@ -22,6 +22,11 @@ export { getCampaign } from "./get-campaign";
 
 export { castVote } from "./cast-vote";
 
+
+export { updateStats } from "./update-stats";
```

Comment:
I think this should not be there

Reply:
We split updateStats on another PR, i don't know why it's in this one but it should be here